libraries/ESP8266mDNS/src/LEAmDNS.cpp

```diff
@@ -822,6 +827,15 @@ uint32_t MDNSResponder::answerCount(const MDNSResponder::hMDNSServiceQuery p_hSe
     return (pServiceQuery ? pServiceQuery->answerCount() : 0);
 }
 
+std::vector<MDNSResponder::MDNSServiceInfo>  MDNSResponder::answerInfo (const MDNSResponder::hMDNSServiceQuery p_hServiceQuery) {
+    std::vector<MDNSResponder::MDNSServiceInfo> tempVector;
+	for (uint32_t i=0;i<answerCount(p_hServiceQuery);i++)
+    {
+		tempVector.emplace_back(MDNSServiceInfo(*this,p_hServiceQuery,i));

```

Comment:
This should be tempVector.emplace_back(*this, p_hServiceQuery, i);

Reply:
done